src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+@import "../utils/functions";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  flex-shrink: 0;
+  padding: 5px;
+  gap: $gap-size;
+  height: min-content;
+  width: $cell-size * 7 + $gap-size * 6 + 10;
+  background-color: lightblue;
+
+  &__cell {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    @include square($cell-size);
+    border: 1px solid black;
+    transition: 0.5s;
```

Comment:
include property that you use transition for (e.g. transition: background-color 0.5s, transform 0.5s)